javascript/javascript3/week1/giphy.js

```diff
@@ -0,0 +1,43 @@
+//fetch data
+//connect button with eent listener
+//assign img and gif
+//create img and append it the document
+
+const searchGifBtn = document.getElementById("searchGifBtn")
+const gifType = document.getElementById("gifType")
+const noOfGif = document.getElementById("noOfGif")
+
+let searchString;
+let gifNum;
+searchGifBtn.addEventListener("click", getGiphy)
+function getGiphy() {
```

Comment:
Think about how you can get your functions to do one thing only. And how this action must be evident just be reading the name. Usually this means writing small functions. Its called "The single responsibility principle"